libs/ebpfnetsh/programs.cpp

```diff
@@ -201,7 +201,10 @@ handle_ebpf_add_program(
         program.clean();
         return ERROR_SUPPRESS_OUTPUT;
     }
-
+    bool boolean = attach_type == EBPF_ATTACH_TYPE_BIND;
+    fprintf(stderr, "attach_type == EBPF_ATTACH_TYPE_BIND %d", boolean);
```

Comment:
I thought all of these were moving to std::cerr

Reply:
Oh! This code was for helping me trace the error. I accidentally pushed this file. Thanks for catching this. Removed.